src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Description 
+            </th>
+            <th>
+                Ingest parsing 
+            </th>
+            <th>
+                Query time parsing 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                Recommended usage
+            </td>
+            <td>
+                Best for creating permanent attributes you can query in the future
+            </td>
+            <td>
+                Best for doing quick queries on attributes that aren't permanent 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Parsing language                
+            </td>
+            <td>
+                You create grok patterns and regular expressions 
+            </td>
+            <td>
+                New Relic creates queries for you using the NRQL [aparse](/docs/nrql/nrql-syntax-clauses-functions/#func-aparse) function 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Timing
+            </td>
+            <td>
+                Applied at ingest
+            </td>
+            <td>
+                Applied when you query
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Results
+            </td>
+            <td>
+                Makes permanent changes to stored log 
+            </td>
+            <td>
+                Temporarily alters the logs table in the UI
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Live tail logs
+            </td>
+            <td>
+                Live tail logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Live tail logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Exported logs
+            </td>
+            <td>
+                Exported logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Exported logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Number of attributes
+            </td>
+            <td>
+                A maximum of [255 attributes](/docs/logs/log-api/log-event-data/#events) is available at ingest (the actual number attributes you can parse at ingest depends on the nature of your logs)
+            </td>
+            <td>
+                You can parse a maximum of 32 temporary attributes across all rules for query time parsing
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## How to create a query time parsing rule [#how-to] 
+
+Here's an overview of the tasks for creating query time parsing rules:
```

Comment:
What about, "Here's a guide to create query time parsing rules"- tasks and guide have different tones.

Reply:
@homelessbirds I like this! I'll use it!